octopusdeploy_framework/schemas/tenant_common_variable.go

```diff
@@ -0,0 +1,29 @@
+package schemas
+
+import (
+	"github.com/OctopusDeploy/terraform-provider-octopusdeploy/octopusdeploy_framework/util"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+)
+
+const (
+	TenantCommonVariableResourceDescription = "Tenant Common Variable"
+	TenantCommonVariableResourceName        = "tenant_common_variable"
+)
+
+func GetTenantCommonVariableResourceSchema() schema.Schema {
+	return schema.Schema{
+		Description: "Manages a tenant common variable in Octopus Deploy.",
+		Attributes: map[string]schema.Attribute{
+			"id":                      util.GetIdResourceSchema(),
+			"space_id":                util.GetSpaceIdResourceSchema(TenantCommonVariableResourceDescription),
+			"tenant_id":               util.GetRequiredStringResourceSchema("The ID of the tenant."),
+			"library_variable_set_id": util.GetRequiredStringResourceSchema("The ID of the library variable set."),
+			"template_id":             util.GetRequiredStringResourceSchema("The ID of the variable template."),
+			"value": schema.StringAttribute{
+				Required:    true,
```

Comment:
The SDKv2 resource was `Optional: true` so assume this should also be optional.

The old resource also had a default value, not sure if that's needed here but I think it'd be `nil` otherwise.